units/en/unit5/run-in-notebook.mdx

```diff
@@ -0,0 +1,156 @@
+# Run in notebook
+
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://githubtocolab.com/dylanebert/ml-for-3d-course-notebooks/blob/main/capstone.ipynb)
+
+The first open is to run in a Colab notebook. This is the easiest way to validate the code quickly.
+
+## Setup
+
+1. Click the "Open in Colab" button above.
+2. Change the runtime type to GPU.
+3. Scroll down to the `Run in this notebook section`.
+
+## Run the demo
+
+Start by installing dependencies.
+
+```bash
+!pip install -r https://huggingface.co/spaces/dylanebert/LGM-tiny/raw/main/requirements.txt
+```
+
+Then, run the demo code. This is exactly the same as in the space `app.py`. To ensure your model is working as expected, replace both instances of `dylanebert/LGM-full` with your `{username}/{model_name}`. Then, run the code.
+
+```python
+import shlex
+import subprocess
+
+import gradio as gr
+import numpy as np
+import spaces
```

Comment:
Do we need this dependency? we can simplify this without the `spaces` package stuff (so that people focus on the space instead of black magic packages)